extensions/amp-vk/validator-amp-vk.protoascii

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-vk
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-vk"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    requires_usage: DEPRECATED_WARNING
+    deprecated_allow_duplicates: true
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-vk>
+  html_format: AMP
+  tag_name: "AMP-VK"
+  requires_extension: "amp-vk"
+  attrs: {
+    name: "data-embedtype"
+    mandatory: true
+    value_regex_casei: "(post|poll)"
+  }
+  attrs: {
```

Comment:
no need to list `data-` unless they are required or need regex, so you can remove these ones.

Reply:
thanks, removed these